src/components/Calculator.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 import KeyButtons from './keys';
-import './Calculator.css';
+import './calculator.css';
```

Comment:
- [OPTIONAL] Please kindly correct the spellings of your `Calculator.css` file, the first letter should be in caps if not your app generate an error.

Reply:
make changes to all CSS files as first letter caps.